pkg/process/checks/container.go

```diff
@@ -63,6 +63,10 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 	start := time.Now()
 	ctrList, err := util.GetContainers()
 	if err != nil {
+		if err == containercollectors.ErrPermaFail || err == containercollectors.ErrNothingYet {
+			log.Debug("container collector was not detected, container check will not return any data")
```

Comment:
I think this one should be `log.Trace` as well to be honest - either that or find a way to stop emitting the error after the first one.

Reply:
I disagree, but don't feel too strongly. We rarely ask users to enable trace logging, even if the agent is having issues. And a debug log every 10 seconds isn't all that frequent (for debug logs).

Reply:
How was this check even enabled if we were unable to detect a container collector...?

Reply:
We started enabling it all the time as part of https://github.com/DataDog/datadog-agent/pull/3958

Reply:
I don't see the part there that automatically starts the containers check - perhaps I'm missing it?

We should only queue the `connections` check if the system-probe is enabled (and only insert the `containers` check if it's actually enabled)